cibuildwheel/frontend.py

```diff
@@ -42,7 +42,11 @@ def _get_verbosity_flags(level: int, frontend: BuildFrontendName) -> list[str]:
             return ["-" + level * "v"]
         if level < 0:
             return ["-" + -level * "q"]
-    elif not 0 <= level < 2:
+    elif level > 1:
+        return ["-v"]
+    elif level < 0:
+        if frontend not in {"build", "build[uv]"}:
+            return ["-q"]
```

Comment:
```suggestion
```

Reply:
This was there to support #2322, but I guess I can re-add it.